src/generator/AutoRest.Java.Azure/Templates/AzureMethodGroupTemplate.cshtml

```diff
@@ -49,6 +49,10 @@ public class @(Model.MethodGroupImplType)@(Model.ParentDeclaration) {
     {
     @:@(Include(new PagingMethodTemplate(), method))
     }
+    else if (method.SimulateAsPagingOperation)

```

Comment:
In case list operation is not supporting paging, we still want to return Observable<Page> rather than Observable<List>

Fixed https://github.com/Azure/azure-sdk-for-java/pull/1478/files

search for public Observable<Page<StorageAccountInner>> listAsync() {